content/docs/command-reference/pull.md

```diff
@@ -37,17 +39,18 @@ remote.
 
 With no arguments, just `dvc pull` or `dvc pull --remote <name>`, it downloads
 only the files (or directories) missing from the workspace by searching all
-[DVC-files](/doc/user-guide/dvc-file-format) currently in the
+stages in [`dvc.yaml`](/doc/user-guide/dvc-file-format) or
+[`.dvc`](/doc/user-guide/dvc-file-format) files currently in the
 <abbr>project</abbr>. It will not download files associated with earlier commits
 in the <abbr>repository</abbr> (if using Git), nor will it download files that
 have not changed.
 
-The command `dvc status -c` can list files referenced in current DVC-files, but
-missing in the <abbr>cache</abbr>. It can be used to see what files `dvc pull`
-would download.
+The command `dvc status -c` can list files referenced in current `.dvc` files,
+but missing in the <abbr>cache</abbr>. It can be used to see what files
```

Comment:
...current `.dvc` files or staged (found in `dvc.yaml`), but missing in the <abbr>cache...